include/mxnet/c_predict_api.h

```diff
@@ -43,8 +43,6 @@ extern "C" {
 
 /*! \brief manually define unsigned int */
 typedef uint32_t mx_uint;
-/*! \brief manually define 64-bit int */
-typedef int64_t mx_int64;
 /*! \brief manually define float */
 typedef float mx_float;
```

Comment:
Why do we still need this?

Reply:
yes we do other language bindings use this

Reply:
Which language bindings use this?

Reply:
java, C++, R, Scala